src/components/app/App.js

```diff
@@ -55,6 +56,7 @@ class App extends React.Component {
           }
           {!user.loggedIn && <Redirect to="/" />}
         </main>
+        <ReduxToastr />
```

Comment:
Not sure it matters so much but have done anyway as we dont want them displaying over the header.